shared/routes/Repl/Viewer.html

```diff
@@ -17,10 +19,27 @@
 </div>
 
 <style>
+	.iframe-container {
+		border-top: 1px solid #ccc;
+	}
+
 	iframe {
 		width: 100%;
-		height: 100%;
+		height: calc(100vh - 3em);
 		border: none;
+		background-color: white;
```

Comment:
This declaration determines the colour of the iframe, not its contents. If FF users have a dark theme, presumably the background colour of the child's body would be overridden anyway?